docs/conf.py

```diff
@@ -84,8 +84,8 @@
 autodoc_default_options = {
     "exclude-members": (
         "__repr__, __weakref__, __metaclass__, __init__, __format__,  __new__, __str__, __dir__,"
-        "model_config, model_fields, model_post_init, model_computed_fields,"
-        "__ape_extra_attributes__,"
+        "model_config, model_dump, model_fields, model_post_init, model_computed_fields,"
+        "__ape_extra_attributes__, __get_item__, __contains__, __iter__, __len__, __eq__, __hash__,"
```

Comment:
Occasionally we override `__iter__` and `__len__` and `__eq__` and provider docs for these methods, so we don't always want to exclude them